orangecontrib/text/widgets/tests/test_owpreprocess.py

```diff
@@ -0,0 +1,15 @@
+from Orange.widgets.tests.base import WidgetTest
+from orangecontrib.text.corpus import Corpus
+from orangecontrib.text.widgets.owpreprocess import OWPreprocess
+
+
+class TestConcordanceWidget(WidgetTest):
+    def setUp(self):
+        self.widget = self.create_widget(OWPreprocess)
+        self.corpus = Corpus.from_file('deerwester')
+
+    def test_set_corpus(self):
+        """
+        Just basic test.
+        """
+        self.send_signal("Corpus", self.corpus)
```

Comment:
If we are introducing these tests, should we also add some functionality checks? At least some simple ones. For example, we could check that corpus didn't have tokens (`self.assertFalse(self.corpus.has_tokens())`) on input and that the corpus on the output has them (`self.assertTrue(output.has_tokens())`).